### ISSUE TITLE
Avance

### ISSUE THREAD

**Issue by mariagonthier** [IIC2233/syllabus#295]
Hola! No entiendo mucho cómo se evalúa el avance del programa/código. ¿Existe una pauta general, o recomendación? ¿Depende cada caso?
Gracias :)

**Comment by aljara97** [comment]
Hola @mariagonthier 

El objetivo del avance de la tarea es tener una idea a través de un diagrama de clases, de cómo están modelando sus programas. Luego de esto se entregará un feedback general, con el que podrán guiarse y corregir errores de modelación. No hay una pauta para esto, por tanto, te sugiero leer con atención el enunciado en la parte de entidades, para entender el cómo se relacionan cada una de las clases en términos de herencia y agregación. Además de apoyarte en los contenidos del curso.

Suerte 🧙

**Comment by rodrigobj99** [comment]
Es obligatorio subir un avance?

**Comment by aljara97** [comment]
Hola @rodrigobj99 !!

No es obligatorio subir un avance, sin embargo, en caso de que lo hagas puedes **optar hasta a dos décimas** en la nota de la tarea. Recuerda que para la entrega de la tarea, se considerará un **diagrama final** el cual tendrá un puntaje asociado.

**Comment by mariagonthier** [comment]
Es decir, solo hay que subir el diagrama? No el codigo????

**Comment by aljara97** [comment]
@mariagonthier, efectivamente, solo es necesario que subas el diagrama. Sin embargo, el que subas tu código puede permitir que entendamos mejor tu diagrama y detectar errores en caso de que hubieran.

**Comment by camolina2** [comment]
En el enunciado salía que también hay que hacer una explicación del diagrama en formato .md  dónde se sube este? Mi pregunta específica es ¿Subo el diagrama en formato .png y la explicación en formato .md a mi repositorio en la carpeta de tarea 1?

**Comment by aljara97** [comment]
Hola @camolina2, exacto, debes subir ambos documentos a la carpeta `T01` en tu repositorio.
